src/Centreon/Domain/PlatformInformation/Model/PlatformInformationFactory.php

```diff
@@ -54,6 +54,9 @@ public function createRemoteInformation(array $information): PlatformInformation
         $platformInformation = new PlatformInformation($isRemote);
```

Comment:
```suggestion
        $platformInformation = new PlatformInformation(true);
```
And remove the line above `$isRemote = true;`

Reply:
You could maybe refactor public function createCentralInformation in this way :
```
public function createCentralInformation(): PlatformInformation
    {
       return new PlatformInformation(false);
    }
```